### PR TITLE
Add preference to Media Profiles

### LINKED ISSUE #31: Media Profiles and Prefered Mode
One important attribute in an EDID is the prefered mode. We have documented this in bcp-005-01 but it does not carry over to the Media Profiles.

When a `PUT /media-profiles` occurs how does the sender know which corresponds to the "prefered mode"?

### PR DESCRIPTION
**Pull request by N-Nagorny**
Resolves #31.

### CODE REVIEW

**Comment by cristian-recoseanu** [comment]
Don't we need to have preference for both the Video and Audio variant of the media profile?

**Comment by N-Nagorny** [comment]
I didn't find preferred audio modes in BCP-005-01 and the EDID specification.

**Review by prince-chrismc — APPROVED** [review]

**Comment by cristian-recoseanu** [comment]
So just to clarify does the preference property in the Media Profile not get filled in from receiver capabilities? BCP-004-01 supports that for anything. So we could have audio receivers which have preferences in receiver capabilities. Is this saying we should ignore preference from audio receivers?

File: APIs/schemas/media_profile_audio.json [hunk 1]
```diff
@@ -2,6 +2,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "properties": {
+    "preference": {
+      "type": "number",
```
Comment (prince-chrismc) [inline APIs/schemas/media_profile_audio.json:6]:
```suggestion
      "type": "number",
      "default": 0,
```

**Review by prince-chrismc — COMMENTED** [review]
From our call

File: APIs/schemas/media_profile_video.json [hunk 2]
```diff
@@ -2,6 +2,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "properties": {
+    "preference": {
+      "type": "number",
```
Comment (N-Nagorny) [inline APIs/schemas/media_profile_video.json:6]:
```suggestion
      "type": "number",
      "default": 0,
```

**Pull request merged**